tests/src/Functional/ContentProjectRenderTest.php

```diff
@@ -291,4 +320,102 @@ protected function getStorage(string $entity_type_id): EntityStorageInterface {
     return \Drupal::entityTypeManager()->getStorage($entity_type_id);
   }
 
+  /**
+   * Updates the project date, saves the node, and refreshes the page.
+   *
+   * @param \Drupal\node\NodeInterface $node
+   *   Node to update.
+   * @param string $begin
+   *   Start date string.
+   * @param string $end
+   *   End date string.
+   */
+  protected function setProjectDateRange(NodeInterface $node, string $begin, string $end): void {
+    $node->oe_project_dates = [
+      'value' => (new DrupalDateTime($begin, 'Europe/Brussels'))->format('Y-m-d'),
+      'end_value' => (new DrupalDateTime($end, 'Europe/Brussels'))->format('Y-m-d'),
+    ];
+    $node->save();
+    $this->drupalGet($node->toUrl());
+  }
+
+  /**
+   * Asserts that the d-none class has been removed from project status.
+   */
+  protected function assertProjectStatusRevealed(): void {
+    $status_section = $this->assertSession()->elementExists('css', '.bcl-project-status');
+    $this->assertFalse($status_section->hasClass('d-none'));
+  }
+
+  /**
+   * Asserts start and end timestamp in project status section.
```

Comment:
```suggestion
   * Asserts 'start' and 'end' timestamp in project status section.
```

Reply:
Not sure. The quotes imply that this refers to existing keys or machine names somewhere. Which is not really the case.

Reply:
mm, ok